database/database.go

```diff
@@ -87,7 +87,7 @@ func ConnectPostgresqlDatabase(connectionString, password string) (core.Metadata
 
 func createRepository(dialector gorm.Dialector) (core.MetadataDatabase, error) {
 	db, err := gorm.Open(dialector, &gorm.Config{
-		Logger:                 logger.Default.LogMode(logger.Warn),
+		Logger:                 gormzerolog.Logger{},
```

Comment:
Seems like a lot of whitespace here?

Reply:
It's what gofmt wants because of the longer field name below.